boards/arm/nrf52840_pca10090/integrity.c

```diff
@@ -49,3 +49,14 @@ BUILD_ASSERT_MSG(IS_ENABLED(CONFIG_BOARD_PCA10090_SWITCH0_PHY) ||
 BUILD_ASSERT_MSG(IS_ENABLED(CONFIG_BOARD_PCA10090_SWITCH1_PHY) ||
 		 IS_ENABLED(CONFIG_BOARD_PCA10090_SWITCH1_ARDUINO),
 		 "Invalid switch 2 routing");
+
+#if CONFIG_BOARD_PCA10090_NRF52840_RESET
```

Comment:
This could be checked inside the `BUILD_ASSERT_MSG` as `!IS_ENABLED(CONFIG_BOARD_PCA10090_NRF52840_RESET)`. You won't need then to add this missing newline at the end of the file. ;-)

Reply:
I don't think that would work, I would need to negate all the other existing `IS_ENABLED()` which I think makes it more confusing. I will leave as-is but add the newline :)

Reply:
Fixed